javascript/lib/api/src/model/model.ts

```diff
@@ -139,10 +134,9 @@ export abstract class IndexedEntityModel<T extends IndexedEntityModel<T, V>, V>
    *
    * @returns The object representation of the Entity.
    */
-  toObject(): Object | undefined {
-    if (this._elements) {
-      // @ts-ignore
-      return IndexedEntityModel.toPlainObject(this._elements, element => element.toObject());
+  static toObject<T extends EntityModel>(entityModel: IndexedEntityModel<T>): Object | undefined {
```

Comment:
If this method would accept the `entityModel` to also be `undefined`, we could get rid of all the `undefined` checks in each model class (e.g. `this.subjects !== undefined ? Subjects.toObject(this.subjects) : undefined`)